content/en/Getting started/details.md

```diff
@@ -32,8 +32,15 @@ Our pentests share the characteristics listed in this section:
 
 ### Network Information
 
-Our pentesters send requests from one or more of IP addresses on a Virtual
-Private Network. We'll share these addresses when you create an actual pentest.
+Our pentesters send requests from one or more IP addresses on a Virtual
+Private Network. Share this pentest with your network administrator. They may need
```

Comment:
'Share this pentest' seems to be a mistake and should be 'Share this list' or something along those lines.

EDIT: nevermind, I see why it was worded that way now